std_msgs/Header header

float64 time

#[m/s]
float32 speed

#[mm/s]
uint16  speed_rms

uint8   flags
